profiles/common/modules/features/nexteuropa_editorial/nexteuropa_editorial.install

```diff
@@ -268,3 +256,77 @@ function nexteuropa_editorial_update_7002() {
   // WYSIWYG tracking settings for "Published" status case.
   variable_set('nexteuropa_editorial_tracking_status', 1);
 }
+
+/**
+ * NEPT-202: Enable nexteuropa_trackedchanges module.
+ *
+ * Update enables the module only if tracked changes are detected and
+ * migrate nexteuropa_editorial settings set here in the module to enable.
+ */
+function nexteuropa_editorial_update_7004(&$sandbox) {
+
+  if (!isset($sandbox['progress'])) {
+    $sandbox['progress'] = 0;
+    $field_types = array(
+      'text_long',
+      'text_with_summary',
+      'text',
+    );
+    $field_info_map = field_info_field_map();
+    $filtered = array_filter($field_info_map, function ($item) use ($field_types) {
+      return in_array($item['type'], $field_types);
+    });
+
+    $sandbox['field_list'] = array_keys($filtered);
+
+    $sandbox['max'] = count($sandbox['field_list']);
+  }
+
+  $field_name = array_shift($sandbox['field_list']);
+
+  $query = db_select('field_revision_' . $field_name, 'fr');
+  $query->where('fr.' . $field_name . '_value REGEXP \'<span[^>]+class="[^"]*ice-[^>]+>\'')
+        ->condition('fr.deleted', 0);
+  $query->addExpression('1');
+  $query->distinct(TRUE);
+  $scanning_results = $query->execute()->fetchField();
+
+  if ($scanning_results) {
```

Comment:
This is problematic in case there is no content on the site you are updating or no content recently used track changing (I've tested an upgrade of a clean platform installation and I have spotted this problem).

Maybe we should change the criteria into "if at least one text format has track changed enabled" instead of relying on content?